tests/integration/schema_registry/test_json_serializers.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+from confluent_kafka.cimpl import TopicPartition
+
+from confluent_kafka.schema_registry.json_schema import JsonSerializer, JsonDeserializer
```

Comment:
If someone is using JSON and not Avro, will they be required to install any avro dependencies? 

(i think no, but just checking)

Reply:
Currently yes, that said the Avro package "extra" has caught a few folks off guard though. If we think its a huge deal we can make it so you have to specify each dependency in the extras. i.e. pip install confluent-kafka[avro, protobuf, jsonschema]. I think it gets a bit weird when you have dependencies for your dependencies though. I.e. I'm not sure how you would handle pip install confluent-kafka[schema_registry, avro, protobuf, jsonschema]. It probably works but it starts to make for a sort of annoying install. Anyway open to  the idea if we think its important.

Reply:
in .net i have 5 separate packages: https://github.com/confluentinc/confluent-kafka-dotnet/tree/1.4.x#referencing

i think it's important - dependencies can often cause headaches, and it often pays to have a strong preference to avoiding additional ones unless absolutely necessary.

Reply:
That's fair it just feels a bit clumsy to me for a scripting language because the source is there and in executable form but the dependencies are not. That said it's easy enough to make extras and have them all require the schema registry dependencies. I'll add it tomorrow.

Reply:
I think this is a bit bigger than this PR; let's tackle afterwards. The problem I have with the current setup is that all of this code is available to your application when installing confluent-kafka even without the dependencies. This means it won't  be caught until runtime which is sloppy. Instead we should move these to their own top-level packages which can be referenced as extras when installing confluent-kafka. 

So confluent-kafka-avro, confluent-kafka-json, confluent-kafka-schema-registry would all be their own projects which could be optionally installed with extras from the main confluent-kafka package. The serializer packages would list confluent-kafka-schema-registry as a dependency insuring its installed along with the serializer. 

With this setup installation would still look like this `pip install confluent-kafka[avro, json]`. Except none of these packages would be available for import until explicitly installed.

Reply:
i'm mostly concerned about backwards compatibility. but if this is all considered "experimental" as an extension of the `SerializingProducer` being marked as such, then that's fine.

Reply:
fixed